### PR TITLE
[OP-1] deployment info added to README

### PR DESCRIPTION
**Pull request by macfarla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,13 +24,20 @@ The Dapp will facilitate managing permissioning rules and maintaining the list o
...
 1. Start a terminal session and start a truffle Ganache node running `truffle develop`. This will start a Ganache node and create a truffle console session.
 1. In the truffle console, run all migrations from scratch with `migrate --reset`. Keep this terminal session open to maintain your Ganache node running.
 1. In a new terminal session, navigate to the `app/` directory and run `npm install` to install all required dependencies and `npm start` to start the web server that is serving our dapp. You only need to run `npm install` if you are running the app for the first time.
 1. In your browser, connect Metamask to the Ganache network (the default endpoint is `http://127.0.0.1:9545/`)
 1. When you start Ganache, it gives you a list of accounts and private keys. Import the first one in Metamask to impersonate the first admin of the system.
 1. Navigate to `http://localhost:3000` to access the Permissioning Dapp.
 1. All changes made to the smart contracts or to the dapp code are automatically refreshed on the website. There is no need to restart the web server after making changes.
+
+To deploy the permissioning Dapp:
```
Comment (mark-terry) [inline README.md]: Do these instructions tell you how to deploy the Dapp, or build it?

File: README.md [hunk 2]
```diff
@@ -24,13 +24,20 @@ The Dapp will facilitate managing permissioning rules and maintaining the list o
...
+
+1. After you have migrated the contracts (as above)
```
Comment (mark-terry) [inline README.md]: This step doesn't contain an action - should it be a precondition?

File: README.md [hunk 3]
```diff
@@ -24,13 +24,20 @@ The Dapp will facilitate managing permissioning rules and maintaining the list o
...
+1. `npm run build` will assemble index.html and all other files in `build/`
+1. You can use your preferred web server technology to serve the contents of build/ as static files.
```
Comment (lucassaldanha) [inline README.md]: Is there value in adding an example using a simple http server (node has got some options) or even apache?

File: README.md (hunk 1 above)
Comment (lucassaldanha) [reply README.md]: They are all about building for deployment (the end artifact is a set of static files that should be served by an http server).

File: README.md [hunk 4]
```diff
@@ -24,13 +24,25 @@ The Dapp will facilitate managing permissioning rules and maintaining the list o
...
+1. In a new terminal session, navigate to the `app/` directory 
+1. Run `npm install` to install all required dependencies and `npm start` to start the web server that is serving our dapp. You only need to run `npm install` if you are running the app for the first time.
 1. In your browser, connect Metamask to the Ganache network (the default endpoint is `http://127.0.0.1:9545/`)
 1. When you start Ganache, it gives you a list of accounts and private keys. Import the first one in Metamask to impersonate the first admin of the system.
 1. Navigate to `http://localhost:3000` to access the Permissioning Dapp.
 1. All changes made to the smart contracts or to the dapp code are automatically refreshed on the website. There is no need to restart the web server after making changes.
+
+#### Build the permissioning Dapp for deployment ####
+
+1. First [Compile and migrate the contracts](#compile-and-migrate-the-contracts)
```
Comment (mark-terry) [inline README.md]: Is 'First' superfluous, given it's 1. in the list?

File: README.md (hunk 3 above)
Comment (macfarla) [reply README.md]: I think someone following these instructions would know how to do that

File: README.md (hunk 4 above)
Comment (macfarla) [reply README.md]: removed

**Pull request merged**